lib/view_models/download_view_model.dart

```diff
@@ -63,15 +100,15 @@ class DownloadViewModel extends StateNotifier<DownloadState> {
       _logger.e('Error fetching downloaded videos: $e');
     }
   }
+
   Future<void> deleteDownload(int videoId) async {
```

Comment:
a small adjustment, before deleting a downloaded video, we can show the user a dialogue card like "Are you sure you want to delete?" or similar. because I can delete the video directly and it is not very optimal in terms of user experience